third-party/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ project(NVIM_DEPS)
 include(CheckCCompilerFlag)
 
 # Point CMake at any custom modules we may ship
-list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
+list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake" "${PROJECT_SOURCE_DIR}/../cmake")
```

Comment:
if user has `cmake/` directory next to the `neovim/` directory could this accidentally add that?

LGTM to me anyways, if it helps make other stuff easier.